### PR TITLE
Making the 'RestoreSources' configurable: CliCommandLineParser 

### PR DESCRIPTION
**Pull request by ghost**
Delete - NuGet.config
Set 'RestoreSources' in RestoreSources.props - import to: Directory.Build.props

Unrelated:
README.md -formatting
tools/PushToBlobFeed.targets - helpful comment

@dotnet/dotnet-cli
@jcagme
@karajas
@chcosta
@weshaggard
@mmitche 
@nguerrera

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -2,6 +2,11 @@
     <Import Condition="Exists('obj/CommitCount.props')"
             Project="obj/CommitCount.props" />
     <PropertyGroup>
-        <CommitCount Condition="'$(CommitCount)' == ''">0</CommitCount>
+      <CommitCount Condition="'$(CommitCount)' == ''">0</CommitCount>
+      <RestoreSources>
```
Comment (livarcocc) [inline Directory.Build.props]: Should this be straight in the Directory.Build.Props? I think it should probably be in its own props file.

Comment (johnbeisner) [reply Directory.Build.props]: Done...

File: RestoreSources.props [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<Project ToolsVersion="15.0">
+  <PropertyGroup>
+    <RestoreSources Condition="'$(RestoreSources)' == ''">
```
Comment (weshaggard) [inline RestoreSources.props]: Just out of curiosity do you expect RestoreSources to be set anywhere else? For most of our core repo's we have something that looks like https://github.com/dotnet/corefx/blob/master/dir.props#L74.

File: RestoreSources.props [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
+      https://api.nuget.org/v3/index.json
```
Comment (weshaggard) [inline RestoreSources.props]: Usually a good idea to put a `;` after each entry to ensure any concatenation elsewhere works as expected. It is better to have more then less in this case.

Comment (johnbeisner) [reply RestoreSources.props]: Yup, sounds good...

File: RestoreSources.props (hunk 2 above)
Comment (johnbeisner) [reply RestoreSources.props]: Thank you for the coding tips.

**Review by mmitche — APPROVED** [review]

**Pull request merged**